docs/tutorials/setting-up-an-nfs-file-share.md

```diff
@@ -97,56 +97,54 @@ After creating the file, the `nfsd` daemon [should automatically start up][OSXSe
 
 If you make changes to `/etc/exports`, activate them with `nfsd update`.
 
-Now update the fstab on the EV3.  
-
 ## How To Do It - EV3
 
-On the EV3 we first need to enable and start NFS modules. Type these commands on the command line:
-
-    sudo systemctl enable nfs-common.service
-    sudo systemctl start nfs-common.service
-    sudo systemctl enable rpcbind.service
-    sudo systemctl start rpcbind.service
-
-Next you'll need to update a file (as root) called `/etc/fstab`. You should have already set up USB Networking, so `ssh` to the EV3 and run an editor like `vi` or `nano` to edit the file. Here's the line you want to add to `/etc/fstab` - DO NOT TOUCH ANYTHING ELSE IN THERE!
-
+On the client - ev3dev - side, we need to create a system mount unit in order to mount our newly created NFS share. **Note**: The classic way of mounting the NFS share via an entry in /etc/fstab does not work!
 
-    # NOTE - the following examples all use the same IP address for the host, in practice, there would
-    #        be separate addresses for each host!
+First off, we need the NFS kernel module. Open an SSH connection to ev3dev and type the following:
 
-    # For the Linux example, it would look like:
-    192.168.2.1:/home/hostuserid/nfs/ev3dev /home/robot/nfs/linux   nfs users,noauto,rw,vers=3  0 0
+    modprobe nfs
+    
+To make this persistent across reboots, we need to add NFS to the module file:
 
-    # For the Windows Hanewin example, it would look like:
-    192.168.0.199:\E\Users\James\Dropbox\ev3dev                       /home/robot/nfs/windows nfs users,noauto,rw,vers=3  0 0
+    echo NFS >> /etc/modules
```

Comment:
This also needs to be run as root. (or possibly could be omitted like I said above)

    sudo sh -c 'echo nfs >> /etc/modules'